src/components/step-sequence/step-sequence-item/step-sequence-item.js

```diff
@@ -0,0 +1,35 @@
+import Icon from 'components/icon';
```

Comment:
this needs to be a relative path, otherwise it won't work when it is packaged up in a tag